skore/src/skore/sklearn/train_test_split/train_test_split.py

```diff
@@ -140,10 +146,15 @@ class labels.
         shuffle=shuffle,
         stratify=stratify,
         y=y,
+        y_test=y_test,
+        y_labels=y_labels,
         ml_task=ml_task,
     )
 
-    for warning_class in [HighClassImbalanceWarning]:
+    for warning_class in [
+        HighClassImbalanceWarning,
+        HighClassImbalanceTooFewExamplesWarning,
+    ]:
```

Comment:
I'd like to refactor this to something like
```suggestion
    for warning_class in all_warnings:
```
@thomass-dev Would you know how to do this?

Reply:
Set in `warning.__init__.py` a constant variable :

```python
WARNINGS = [
    HighClassImbalanceWarning,
    HighClassImbalanceTooFewExamplesWarning,
]
```

Reply:
Or more complex but dynamically (i don't recommend), you can import `train_test_split.warning`, check for all its attributes and iterate over the one inheriting from `TrainTestSplitWarning`.

Reply:
done, thanks!